aws-lc-sys/builder/cmake_builder.rs

```diff
@@ -101,6 +104,15 @@ impl CmakeBuilder {
         cmake_cfg.define("DISABLE_PERL", "ON");
         cmake_cfg.define("DISABLE_GO", "ON");
 
+        if Some(true) == env_var_to_bool("AWS_LC_SYS_NO_ASM") {
+            let opt_level = cargo_env("OPT_LEVEL");
+            if opt_level == "0" {
+                cmake_cfg.define("OPENSSL_NO_ASM", "1");
+            } else {
+                panic!("AWS_LC_SYS_NO_ASM only allowed for debug builds!")
```

Comment:
What are we accomplishing then? It seems disingenuous to artificially increase platform support with this option, only allow it for debug builds?

Reply:
I see the goals that this achieves as being: (1) Facilitating improved debugging/tracing of cryptographic operations at lower levels, and (2) Unblocking use of our library in development environments on platforms we don't (or can't) fully support currently (such as Windows/ARM64).